tests/data/deepglobelandcover/data.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+
+import numpy as np
+from PIL import Image
+from torchvision.datasets.utils import calculate_md5
+
+
+def generate_test_data(root: str, n_samples: int = 3) -> str:
+    """Creates test data archive for DeepGlobeLandCover dataset and
+    returns its md5 hash.
+    Args:
+        root (str): Path to store test data
+        n_samples (int, optional): Number of samples.
+    Returns:
+        str: md5 hash of created archive
+    """
```

Comment:
```suggestion
    """Create test data archive for DeepGlobeLandCover dataset.
    
    Args:
        root: path to store test data
        n_samples: number of samples.
        
    Returns:
        md5 hash of created archive
    """
```
We don't run `pydocstyle` on these files but if we did it would require these changes